channels/in.m3u

```diff
@@ -381,7 +381,7 @@ http://103.199.160.85/Content/puthuyugam/Live/Channel(Puthuyugam)/index.m3u8
 #EXTINF:-1 tvg-id="RPlusNews.in" tvg-name="R Plus News" tvg-country="IN" tvg-language="" tvg-logo="" group-title="News",R Plus News (360p)
 https://cdn.smartstream.video/smartstream-us/rplus/rplus/playlist.m3u8
 #EXTINF:-1 tvg-id="RajDigitalPlus.in" tvg-name="Raj Digital Plus" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="Movies",Raj Digital Plus (404p)
-http://198.144.149.82:8080/NOTV/RELAXTV/index.m3u8?token=GTRajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
+http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls
```

Comment:
Same thing.
![image](https://user-images.githubusercontent.com/7253922/130042388-1b4e1aba-fe73-4e7b-bf3b-df92b3f809cc.png)

Reply:
this is an old link
```suggestion
http://acv.asianetmobiletvplus.com/channels/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_hls/rajdigital_htfuqchouekpefzxodtbzaojtryxpqkx_master.m3u8
```